### PR TITLE
[DOC-9332] Document new regions on Azure (Part 2 of N, CC-25999)

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-9332] Document new regions on Azure (Part 2 of N, CC-25999)

### CODE REVIEW

File: src/current/_includes/releases/cloud/2024-04-04.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## April 4, 2024
+
+<h3 id="2024-04-04-general-updates"> General updates </h3>
+
+- CockroachDB {{ site.data.products.dedicated }} clusters on Azure can now be created in the following [regions]({% link cockroachcloud/regions.md %}):
+
+    Africa          | `southafricanorth`   | Johannesburg
+    Asia Pacific    | `japaneast`          | Tokyo
+                    | `koreacentral`       | Seoul
+                    | `australiaeast`      | New South Wales
```
Comment (adwittumuluri) [inline src/current/_includes/releases/cloud/2024-04-04.md:10]: we always supported `australiaeast` and `germanywestcentral`, they can be removed from this list.

**Review by adwittumuluri — CHANGES_REQUESTED** [review]
We will not be offering the following regions, which'll need to be removed from this pr:

* `northcentralus`
* `westcentralus`
* `italynorth`
* `westus`
* `israelcentral`
* `brazilsouth`

File: src/current/_includes/releases/cloud/2024-04-04.md (hunk 1 above)
Comment (adwittumuluri) [reply src/current/_includes/releases/cloud/2024-04-04.md:10]: The full list of regions is given by any org in staging right now.

**Review by adwittumuluri — APPROVED** [review]
LGTM!

**Review by rmloveland — APPROVED** [review]
LGTM once the broken table render is fixed

File: src/current/_includes/releases/cloud/2024-04-18.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                    | `koreacentral`       | Seoul
+    Middle East     | `qatarcentral`       | Doha
+                    | `uaenorth`           | Dubai
+                    | `southcentralus`     | Texas
+                    | `westus3`            | Washington
+    Western Europe  | `francecentral`      | Paris
+                    | `norwayeast`         | Oslo
+                    | `polandcentral`      | Warsaw
+                    | `swedencentral`      | Gävle
+                    | `switzerlandnorth`   | Zürich
```
Comment (mdlinville) [inline src/current/_includes/releases/cloud/2024-04-18.md:18]:
```suggestion
    Geographic Area  | Region Name          | Location
    -----------------|----------------------|---------
    Africa           | `southafricanorth`   | Johannesburg
    Asia Pacific     | `japaneast`          | Tokyo
                     | `koreacentral`       | Seoul
    Middle East      | `qatarcentral`       | Doha
                     | `uaenorth`           | Dubai
                     | `southcentralus`     | Texas
                     | `westus3`            | Washington
    Western Europe   | `francecentral`      | Paris
                     | `norwayeast`         | Oslo
                     | `polandcentral`      | Warsaw
                     | `swedencentral`      | Gävle
                     | `switzerlandnorth`   | Zürich
```

**Pull request merged**